apps/studio/src/server/modules/page/page.router.ts

```diff
@@ -287,4 +288,24 @@ export const pageRouter = router({
       await startProjectById(ctx.logger, codeBuildId)
       return addedVersionResult
     }),
+
+  updatePageSettings: protectedProcedure.input(pageSettingsSchema).mutation(
```

Comment:
nit: since we are already in the `page` router, maybe the procedure can be shorter to be less verbose:

```suggestion
  updateSettings: protectedProcedure.input(pageSettingsSchema).mutation(
```

Reply:
going to prefer being abit more explicit here .-.

Reply:
wait nvm, i realised at the callsite it gets destructured to use `mutate` so this isn't directly relevant anyways